bin/reth-bench/src/bench/output.rs

```diff
@@ -145,15 +146,14 @@ pub(crate) struct TotalGasOutput {
 
 impl TotalGasOutput {
     /// Create a new [`TotalGasOutput`] from a list of [`TotalGasRow`].
-    pub(crate) fn new(rows: Vec<TotalGasRow>) -> Self {
+    pub(crate) fn new(rows: Vec<TotalGasRow>) -> eyre::Result<Self> {
         // the duration is obtained from the last row
-        let total_duration =
-            rows.last().map(|row| row.time).expect("the row has at least one element");
+        let total_duration = rows.last().map(|row| row.time).ok_or_eyre("empty results")?;
```

Comment:
should this just use duration zero in this case?

~~unclear when rows can be empty~~

would happen if

this is closed

https://github.com/paradigmxyz/reth/blob/4fcc4457c144a41df443c1b297e58ccbe91735c1/bin/reth-bench/src/bench/new_payload_fcu.rs#L96-L101